lib/web_ui/lib/src/engine/dom.dart

```diff
@@ -225,28 +225,54 @@ extension DomCanvasElementExtension on DomCanvasElement {
   external set width(int? value);
   external int? get height;
   external set height(int? value);
+  external String toDataURL(String? type);
```

Comment:
Is it possible to just make this an optional param, like `toDataURL([String? type])` so that you can use the same syntax as before (i.e. we don't have to explicitly pass `null`)

Reply:
Good catch. I think I noticed at some point that optional parameters don't always work exactly the same with static interop vs the `dart:html`. However, in this case they do seem to be similar. Hopefully my observation was incorrect, and we can continue to match the semantics of the `dart:html` APIs.